chrome/main.js

```diff
@@ -17,8 +17,12 @@ chrome.browserAction.onClicked.addListener(function(tab) {
     console.log('language', language, tab);
     var onInjected = function() {
       console.log('injected');
-      chrome.tabs.sendMessage(tab.id, 'getText', {}, function() {
-         console.log('after getText', arguments);
+      chrome.tabs.sendMessage(tab.id, 'getText', function() {
+        console.log('after getText', arguments);
+    	
+        var wordManager = new WordManager(arguments[0].data, language);
```

Comment:
This eventually needs to grow to traversing the DOM element by element and storing those words in ReadingState - only from reading state it can go to LocalDb(). (It's OK as an intermediate step)